homeassistant/components/zha/number.py

```diff
@@ -278,65 +278,71 @@ async def async_setup_entry(
 class ZhaNumber(ZhaEntity, NumberEntity):
     """Representation of a ZHA Number entity."""
 
-    def __init__(self, unique_id, zha_device, channels, **kwargs):
+    def __init__(
+        self,
+        unique_id: str,
+        zha_device: ZHADevice,
+        channels: list[ZigbeeChannel],
+        **kwargs: Any,
+    ) -> None:
         """Init this entity."""
         super().__init__(unique_id, zha_device, channels, **kwargs)
-        self._analog_output_channel = self.cluster_channels.get(CHANNEL_ANALOG_OUTPUT)
+        self._analog_output_channel = self.cluster_channels[CHANNEL_ANALOG_OUTPUT]
```

Comment:
Slight code change here: if it was optional, then I think all subsequent calls would fail.

Reply:
Shouldn’t be an issue